.circleci/config.yml

```diff
@@ -30,6 +30,8 @@ jobs:
 
       - run: git checkout package-lock.json
 
+      - run: node seeding/script.js
```

Comment:
Sorry, I've never tried integrating seeding with CI. If you don't want to spend time troubleshooting this we can always seed the deployed version manually.